pluma/stream/zeromq.py

```diff
@@ -206,4 +226,30 @@ def __init__(self, eventcode: int, **kw):
                  ('Subject.Position.Y', np.single),
                  ('Point.Position.X', np.single),
                  ('Point.Position.Y', np.single)]],
+            **kw)
+class ProtocolNewSceneStream(ZmqStream):
```

Comment:
```suggestion


class ProtocolNewSceneStream(ZmqStream):
```

Add spacing to separate class definitions for readability.

Reply:
solved in 91d2c97